### PR TITLE
records added

### PR DESCRIPTION
**Pull request by jaxinye**
1111

### CODE REVIEW

File: records [hunk 1]
```diff
@@ -0,0 +1,97 @@
+Directly added a file to the repository. => Push Event
+
+git branch child_1
+git push origin child_1 => One CreateEvent and one PushEvent. {2018-08-12 22:22:54}
+[The PushEvent's 'commits' field is empty, 'created' field is true, 'base_ref' is "refs/heads/master" which is the master branch the child pertaining to.]
+
+git add child1.py
+git commit -m "new file added to child_1 branch"
```
Comment (jaxinye) [inline records:8]: 2222

**Review by jaxinye — COMMENTED** [review]
1111

File: records [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+git push origin child_1 => One PushEvent created. {2018-08-12 22:40:18}
+[The PushEvent's 'commits' field is not empty, comment is in 'commits'=>'message', 'created' field is false, 'base_ref' is null]
+
+git submodule add https://github.com/jaxinye/Python-100-Days.git
+git commit -m "added one submodule"
+git push origin child_1 => One PushEvent created.
+
```
Comment (gitprojecttest1) [inline records:15]: 333

**Review by ghost — CHANGES_REQUESTED** [review]
request change

File: records (hunk 2 above)
Comment (gitprojecttest2) [reply records:15]: 444

**Review by gitprojecttest2 — APPROVED** [review]
approve by 2

File: records (hunk 1 above)
Comment (jaxinye) [reply records:8]: 3333

Comment (jaxinye) [reply records:8]: 4444

File: records (hunk 2 above)
Comment (jaxinye) [reply records:15]: 5555

**Comment by jaxinye** [comment]
comment

**Pull request closed without merging**